server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerScanMetrics.java

```diff
@@ -52,7 +52,7 @@ public void incrementOpenFiles(int numOpenFiles) {
   }
 
   public void decrementOpenFiles(int numOpenFiles) {
-    openFiles.addAndGet(numOpenFiles);
+    openFiles.addAndGet(numOpenFiles > 0 ? numOpenFiles : numOpenFiles * -1);
```

Comment:
So with this change, if the number passed to `decrementOpenFiles()` is positive it will be added to `openFiles` else if its negative, it will be made positive (multiplied by -1) and added to `openFiles`?

Reply:
What you are saying is that I'm doing a really bad job of multi-tasking today. :-)